superset/db_engine_specs/presto.py

```diff
@@ -427,6 +427,7 @@ def _partition_query(  # pylint: disable=too-many-arguments,too-many-locals,unus
         limit: int = 0,
         order_by: list[tuple[str, bool]] | None = None,
         filters: dict[Any, Any] | None = None,
+        is_trino: bool = False,
```

Comment:
If this is the route we want to take, ideally this should be not Trino specific, i.e., maybe something like `use_partition_suffix`.